src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -1041,9 +1041,7 @@ def get_observation(self, event: Event) -> pd.DataFrame:
         pop["tax_year"] = event.time.year - 1
         partners = [
             "Opp-sex spouse",
-            "Opp-sex partner",
```

Comment:
Are these removed b/c a "partner" has no meaning in the context of taxes?

Reply:
Yes. This was in the docs and was incorrectly in here as well. Im guessing it is also you need to be a spouse to be a joint filer but thats a guess.